libswiftnav

```diff
@@ -1 +1 @@
-Subproject commit 2c0d0a776ebdb252ccf94da316c1bf5183b939b0
+Subproject commit 19a3718005fc4c8a8e2fe4a60c381bb5829e34c5
```

Comment:
Yes, this should match the version of libswiftnav, which adds L2C correlator support, that is https://github.com/swift-nav/libswiftnav/pull/315.